src/System.Management.Automation/engine/pipeline.cs

```diff
@@ -688,10 +688,17 @@ private void DoCompleteCore(CommandProcessorBase commandRequestingUpstreamComman
         /// <summary>
         /// Clean up resources for script commands in this pipeline processor.
         /// </summary>
-        internal void Clean()
+        /// <remarks>
+        /// Exception from a 'Clean' block is not allowed to propagate up and terminate the pipeline
+        /// so that other 'Clean' blocks can run without being affected. Therefore, this method will
+        /// catch and handle all exceptions inside, and it will never throw.
```

Comment:
Maybe add where user can find the errors? $error, logs?

Reply:
The code is pretty clear that exception caught here will be written to the error stream. From a user's view, it's the same place where those errors are going to be available, $errors, error variable, redirected error streams, and etc.